tests/suites/test_suite_mpi.data

```diff
@@ -15767,6 +15767,39 @@ mpi_core_montmul:15:15:8:8:"6cfdd6e60912e441d2d1fc88f421b533f0103a5322ccd3f4db84
 mbedtls_mpi_core_montmul #141 (gen)
 mpi_core_montmul:15:15:8:8:"7a09a0b0f8bbf8057116fb0277a9bdf3a91b5eaa8830d448081510d8973888be5a9f0ad04facb69aa3715f00000000000000000000000000000000":"764dec6c05a1c0d87b649efa5fd94c91ea28bffb4725d4ab4b33f1a3e8e3b314d799020e244a835a145ec9800000000000000000000000000000000":"8335616AED761F1F7F44E6BD49E807B82E3BF2BF11BFA6AF813C808DBF33DBFA11DABD6E6144BEF37C6800000000000000000000000000000000051":"36a534b574adac80dfea3dabdf0a8c95f47511ff202eab325e5a6edfb7e3ab28ddc15b2f6023240939ba5b7339f7ada7225376a817263464cfba89d":"7a9f42a7ed20ab041b748b5594d63e5aa5c890cd88826a4120bec4c43a90bfb8c7f78ad9cd91756a5f0f5a9154725b7339f7ada7225376a81726392"
 
+mbedtls_mpi_get_montgomery_constant_unsafe_neg
+mpi_get_montgomery_constant_unsafe_neg:
+
+mbedtls_mpi_get_montgomery_constant_unsafe #1
+mpi_get_montgomery_constant_unsafe:"f":"1":"1"
+
+mbedtls_mpi_get_montgomery_constant_unsafe #2
+mpi_get_montgomery_constant_unsafe:"fd":"ec":"24"
+
+mbedtls_mpi_get_montgomery_constant_unsafe #3
+mpi_get_montgomery_constant_unsafe:"eeff99aa37":"a23bd6a686":"a23bd6a686"
+
+mbedtls_mpi_get_montgomery_constant_unsafe #4
+mpi_get_montgomery_constant_unsafe:"eeff99aa11":"3308cb71":"3308cb71"
+
+mbedtls_mpi_get_montgomery_constant_unsafe #5
+mpi_get_montgomery_constant_unsafe:"800000000005":"6400000000":"6400000000"
+
+mbedtls_mpi_get_montgomery_constant_unsafe #6
+mpi_get_montgomery_constant_unsafe:"7fffffffffffffff":"4":"4"
+
+mbedtls_mpi_get_montgomery_constant_unsafe #7
+mpi_get_montgomery_constant_unsafe:"80fe000a10000001":"5dbc6e833bad575a":"5dbc6e833bad575a"
+
+mbedtls_mpi_get_montgomery_constant_unsafe #8
+mpi_get_montgomery_constant_unsafe:"25a55a46e5da99c71c7":"11637ce1347edeaf669":"1e455bf7451c05bc711"
+
+mbedtls_mpi_get_montgomery_constant_unsafe #9
+mpi_get_montgomery_constant_unsafe:"314dc643fb763f2b8c0e2de00879":"1058ad82120c3a10196bb36229c1":"1058ad82120c3a10196bb36229c1"
+
+mbedtls_mpi_get_montgomery_constant_unsafe #10
+mpi_get_montgomery_constant_unsafe:"8335616aed761f1f7f44e6bd49e807b82e3bf2bf11bfa6af813c808dbf33dbfa11dabd6e6144bef37c6800000000000000000000000000000000051":"78a9f16233856e722242e964006ed8666bfe8e55ea736ea86ce7aa71511e36d9ea1509ad5d821f7777e4a2d885924d15cc11e2ccd85eba69ab04989":"5c9d20a5636b6d7abdec003c1ad87e7c88ebf7238a5d85800d3bc214512cd6269558728307ae94eb389e2ccd85eba69ab0493e8277211ce1be22db"
```

Comment:
This number is around 480 bits long. In general I think it is a good idea to test with longer and longer input. However, I think it would be best to test with lengths that are more interesting for us:
- 192 bits: this will be the minimum length that every system using ECC needs to be able to handle and several actual curves are this long
- 2048 bits: this will be the minimum length that every system using RSA or FFDH will need to support